Recipes/Recipes/RecipeModule/Presenter/RecipePesenter.swift

```diff
@@ -0,0 +1,21 @@
+// RecipePesenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Презентер для экрана с рецептами
+class RecipePresenter {
+    // MARK: - Public Properties
+
+    weak var recipeCoordinator: RecipeCoordinator?
+
+    // MARK: - Private Properties
+
+    private weak var view: UIViewController?
```

Comment:
```suggestion
    private weak var view: UIViewController?
```
вью должно быть абстрактным